test/systems/plonk/placeholder/placeholder.cpp

```diff
@@ -339,6 +339,10 @@ BOOST_AUTO_TEST_CASE(placeholder_permutation_argument_test) {
 
     for (int i = 0; i < argument_size; i++) {
         BOOST_CHECK(prover_res.F[i].evaluate(y) == verifier_res[i]);
+        for (std::size_t j = 0; j < desc.rows_amount; j++) {
+            BOOST_CHECK(prover_res.F[i].evaluate(preprocessed_public_data.common_data.basic_domain->get_domain_element(
```

Comment:
(maybe) add this check to other tests

Reply:
I agree that we need this check in other tests. 
However, it shouldn't be a part of permutation-related PR (imho)